src/API/Site/Controllers/Ads/RecommendationsController.php

```diff
@@ -68,16 +79,18 @@ public function register_routes(): void {
 	 */
 	public function get_collection_params(): array {
 		return [
-			'type' => [
-				'type'        => 'string',
-				'description' => __( 'Filter recommendations by type', 'google-listings-and-ads' ),
-				// This could also use a callback to get the set of supported recommendation types from the `AdsRecommendations` service.
-				'enum'        => [ 'IMPROVE_PERFORMANCE_MAX_AD_STRENGTH' ],
+			'type'        => [
+				'type'        => 'array',
+				'description' => __( 'Filter recommendations by one or more types', 'google-listings-and-ads' ),
+				'items'       => [
+					'type' => 'string',
+					'enum' => $this->allow_recommendations_types,
+				],
 				'required'    => false,
```

Comment:
@mukeshpanchal27 see my other comment, maybe this should be `true` as `type` is now required with no default?